dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/enums/Status.java

```diff
@@ -407,6 +407,8 @@ public enum Status {
 
     NO_CURRENT_OPERATING_PERMISSION(1400001, "The current user does not have this permission.", "当前用户无此权限"),
     FUNCTION_DISABLED(1400002, "The current feature is disabled.", "当前功能已被禁用"),
+    SCHEDULE_TIME_NUMBER(1400003, "The number of complement dates exceed 100.", "补数日期个数超过100"),
```

Comment:
I think we should not limit the date range of user complement. In the actual production environment, it is very common to supplement the time range within one or two years.